tests/test_qb_timeline.py

```diff
@@ -420,3 +421,89 @@ def test_age(self):
         cache_key = QB_StatusCacheKey()
         cache_key.update(hourback)
         assert cache_key.minutes_old() == 60
+
+
+class HasAt(object):
+    """Mock the sorted portion of QBT for testing AtOrderedList"""
+    def __init__(self, at, label):
+        self.at = at
+        self.label = label
+
+
+n = HasAt(datetime.utcnow(), 'now')
+y = HasAt(n.at - relativedelta(days=1), 'yesterday')
+y2 = HasAt(y.at, 'yesterday again')
+wa = HasAt(n.at - relativedelta(weeks=1), 'week ago')
```

Comment:
Please use longer variable names. Unless they get long enough to cause PEP8 nags, longer variable names are easier to grep/find/replace